### DISCUSSION TITLE
Multiplatform application

### DISCUSSION THREAD

**Discussion by gusmanb** [gusmanb/logicanalyzer#15] [Announcements]
We're mostly there, the app is fully functional, only the protocol analyzers need to be finished and test the app in other OS'es.

Here are some screenshots of the app.

![imagen](https://user-images.githubusercontent.com/4086913/179469149-43ac3c25-e11e-4993-81c3-95cae287d752.png)

![imagen](https://user-images.githubusercontent.com/4086913/179469286-b4b2e510-f9ae-404a-9e5a-65f87c59f1a4.png)

![imagen](https://user-images.githubusercontent.com/4086913/179469438-a86abc5c-cf9a-4410-a86c-c163c02bc592.png)

**Comment by X-Ryl669** [comment]
Kudos!

**Comment by gusmanb** [comment]
Protocols ready and analyzer working in Linux 🕺 

![imagen](https://user-images.githubusercontent.com/4086913/179560235-da8bfbf3-8c84-4f77-8434-b01e7d5b3b7f.png)


![imagen](https://user-images.githubusercontent.com/4086913/179559830-f113aae9-5c67-4a19-a121-4bf204477cea.png)

**Comment by gusmanb** [comment]
Been really busy with my job this days so I had no time to finish testing, I want to at least test the app in linux/arm (rpi) before releasing it, I hope to have time this week.

Cheers!

**Comment by moogway82** [comment]
Hi, just wanted to say that I've tried running this for the first time on my Mac OS 10.15 machine and it's working great, used the /dev/tty.usbmodem14101 port and captured fine at 100MHz and 20MHz, including the "Allow more samples" mode. Great project, looking forward to learning to use it more.

Update: Should've said I'm using the 3.5.0.1 release.
